### ISSUE TITLE
Only treat level-1 headings as sections in deck.js backend

### ISSUE THREAD

**Issue by mojavelinux** [asciidoctor/asciidoctor-backends#117]
In the deck.js backend, all section headings are converted to `<section>` HTML elements. Only level-1 headings (two equal signs) should be converted to `<section>` HTML elements. The rest should be converted to regular section headings because they are most likely content.

The workaround currently is to add the `[discrete]` block style to the heading.

**Comment by wimalopaan** [comment]
Well, then you miss the numbering for normal html backend ...

**Comment by mojavelinux** [comment]
Yes, the workaround is less than ideal. Though, you could do something like:

```
ifdef::deckjs[[discrete]]
```

Then pass the `-a deckjs` flag. Again, not ideal, but workable.
